src/test/groovy/org/gradle/android/RoomSchemaLocationWorkaroundTest.groovy

```diff
@@ -42,7 +44,7 @@ class RoomSchemaLocationWorkaroundTest extends AbstractTest {
         buildResult = withGradleVersion(Versions.latestGradleVersion().version)
             .forwardOutput()
             .withProjectDir(temporaryFolder.root)
-            .withArguments("clean", "assemble", "--build-cache", "--stacktrace")
+            .withArguments(CLEAN_BUILD)
             .build()
 
         then:
```

Comment:
should we also make sure that the unitTest tasks came from the cache? Meaning that there wasn't a cache miss caused by room? as well as the androidTestCompile when added.

Reply:
I think so.  Done.